packages/twenty-front/src/hooks/useIsMatchingLocation.ts

```diff
@@ -1,19 +1,25 @@
-import { useCallback } from 'react';
 import { matchPath, useLocation } from 'react-router-dom';
 
 import { AppBasePath } from '@/types/AppBasePath';
 
 export const useIsMatchingLocation = () => {
```

Comment:
For consistency with our codebase, let's do : 

`const { isMatchingLocation } = useIsMatchingLocation()`

A named export allows us to enforce the name of the function everywhere.